setup.py

```diff
@@ -1,17 +1,17 @@
-from setuptools import setup, find_packages
 import os
 import re
-import subprocess
+import subprocess  # noqa: S404
```

Comment:
It's best to accompany lint rule ignores, especially the security-related ones, with code comments explaining why they are okay to have as an exception.